### ISSUE TITLE
Using @responsive with adjacent sibling selector

### ISSUE THREAD

**Issue by mattwaler** [tailwindcss/tailwindcss#947]
**Tailwind Version:** 1.0.2

When trying to create custom subsequent classes via `@responsive`, the outputted CSS is not valid. For example, this css:

```css
@responsive {
  .subseq-border + .subseq-border {
    @apply border-l;
  }
}
```

Results in this CSS:

```css
.subseq-border + .subseq-border {
  border-left-width: 1px;
}
.subseq-border + .sm\:subseq-border {
  border-left-width: 1px;
}
.subseq-border + .md\:subseq-border {
  border-left-width: 1px;
}
```

Is this something that you foresee is fixable?

**Comment by adamwathan** [comment]
This is actually by design currently, the prefix is added to the last class in the selector because that's what made sense for `group-hover`.

Ideally I would actually just have Tailwind throw an error when you try to do this but we wanted it to work for `group-hover` so we ended up where we are now. `@responsive` (and `@variants` as well) are really only intended to be used for simple utility classes, where it's just a single class that does one thing rather than a more complex selector like this.

Maybe one day we can come up with a way to give the user more control over what's generated here but not sure it could be done well without a breaking change 😕

**Comment by adamwathan** [comment]
Closing this for now as a "won't/can't really fix" sort of thing until 2.0 where we can rethink this if necessary.
